.circleci/config.yml

```diff
@@ -163,28 +152,27 @@ jobs:
       - check_release_chaneglog_update_branch
       - merge_target_branch
       - install_chrome
-      - attach_workspace:
-          at: /home/circleci/
+      - install_dependencies
       - run:
           name: Run e2e publication test
           command: |
-            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="publication" --prod_env
+            python -m scripts.run_e2e_tests --deparallelize_terser --suite="publication" --prod_env
```

Comment:
Why are we not skipping the build & install also for this one?

Reply:
First tests in a test grouping need to run install and build so that subsequent tests don't need to. In this case, the "publication" suite is the first test in the "e2e_library_and_collections" grouping, so we don't skip build & install.